meta-mender-tn-imx-bsp/recipes-bsp/u-boot/u-boot-tn-imx/edm-g-imx8mp/0001-configs-edm-g-imx8mp-add-Mender-support.patch

```diff
@@ -24,11 +24,10 @@ index 5d55972019..67f36b0da8 100644
 +CONFIG_SYS_REDUNDAND_ENVIRONMENT=y
 +CONFIG_BOOTCOUNT_LIMIT=y
```

Comment:
If the `BOOTLIMIT` has been removed, shouldn't those two config items also be removed?

Reply:
I used this patch as an example:
https://github.com/TechNexion-customization/meta-mender-community/blob/zeus/meta-mender-tn-imx-bsp/recipes-bsp/u-boot/u-boot-tn-imx/pico-imx8mm/0001-configs-pico-imx8mm-add-Mender-support.patch#L26